vterm-module.c:472:8: error: unused variable ‘following’ [-Werror=unused-variable] bool following = term->height == 1 + pos.row;

When updating `vterm` today within _Emacs 26.3_ in _Ubuntu 18.04_, I got the following error message:

```bash
[ 66%] Built target libvterm
Scanning dependencies of target vterm-module
[ 75%] Building C object CMakeFiles/vterm-module.dir/vterm-module.c.o
/home/dmayr/.emacs.d/elpa/26.3/develop/vterm-20200406.200/vterm-module.c: In function ‘get_col_offset’:
/home/dmayr/.emacs.d/elpa/26.3/develop/vterm-20200406.200/vterm-module.c:238:17: error: unused variable ‘buf’ [-Werror=unused-variable]
   unsigned char buf[4];
                 ^~~
/home/dmayr/.emacs.d/elpa/26.3/develop/vterm-20200406.200/vterm-module.c: In function ‘adjust_topline’:
/home/dmayr/.emacs.d/elpa/26.3/develop/vterm-20200406.200/vterm-module.c:472:8: error: unused variable ‘following’ [-Werror=unused-variable]
   bool following = term->height == 1 + pos.row;
        ^~~~~~~~~
In file included from /home/dmayr/.emacs.d/elpa/26.3/develop/vterm-20200406.200/vterm-module.c:1:0:
/home/dmayr/.emacs.d/elpa/26.3/develop/vterm-20200406.200/vterm-module.c: At top level:
/home/dmayr/.emacs.d/elpa/26.3/develop/vterm-20200406.200/vterm-module.h:105:13: error: ‘refresh_size’ declared ‘static’ but never defined [-Werror=unused-function]
 static void refresh_size(Term *term);
             ^~~~~~~~~~~~
cc1: all warnings being treated as errors
CMakeFiles/vterm-module.dir/build.make:79: recipe for target 'CMakeFiles/vterm-module.dir/vterm-module.c.o' failed
make[2]: *** [CMakeFiles/vterm-module.dir/vterm-module.c.o] Error 1
CMakeFiles/Makefile2:122: recipe for target 'CMakeFiles/vterm-module.dir/all' failed
make[1]: *** [CMakeFiles/vterm-module.dir/all] Error 2
Makefile:100: recipe for target 'all' failed
make: *** [all] Error 2
```

`cmake` version: 3.17 (Kitware PPA)
`libtool-bin` version: 2.4.6

--- a/vterm-module.c
+++ b/vterm-module.c
@@ -235,7 +235,7 @@ static int is_end_of_prompt(Term *term, int end_col, int row, int col) {
 static size_t get_col_offset(Term *term, int row, int end_col) {
   int col = 0;
   size_t offset = 0;
-  unsigned char buf[4];
+
   int height;
   int width;
   vterm_get_size(term->vt, &height, &width);
@@ -468,8 +468,6 @@ static void adjust_topline(Term *term, emacs_env *env) {
   goto_line(env, pos.row - term->height);
   size_t offset = get_col_offset(term, pos.row, pos.col);
   forward_char(env, env->make_integer(env, pos.col - offset));
-
-  bool following = term->height == 1 + pos.row;
 
   emacs_value windows = get_buffer_window_list(env);
   emacs_value swindow = selected_window(env);

--- a/vterm-module.h
+++ b/vterm-module.h
@@ -102,7 +102,7 @@ static void term_flush_output(Term *term, emacs_env *env);
 static void term_process_key(Term *term, emacs_env *env, unsigned char *key,
                              size_t len, VTermModifier modifier);
 static void invalidate_terminal(Term *term, int start_row, int end_row);
-static void refresh_size(Term *term);
+
 void term_finalize(void *object);
 
 emacs_value Fvterm_new(emacs_env *env, ptrdiff_t nargs, emacs_value args[],

